_dashboards/quickstart.md

```diff
@@ -1,60 +1,73 @@
 ---
 layout: default
-title: Quickstart guide
+title: Dashboards quickstart
 nav_order: 2
 has_children: false
 redirect_from:
    - /dashboards/quickstart-dashboards/
 ---
 
-# Quickstart guide
+# Dashboards quickstart
 
-This quickstart guide covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
+This quickstart covers the core concepts that you need to understand to get started with OpenSearch Dashboards. You'll learn how to:
 
 - Add sample data.
-- Explore and inspect data.
+- Explore and interact with data.
 - Visualize data.
 
-Here's a glance at the view you see when you open the **Dashboard** or **Discover** tool.
+Starting with OpenSearch 2.12, you'll see the following homepage view. This redesign considers the OpenSearch community's feedback about simplifying the dashboard experience. Share your [feedback](<insert-link-feedback-dashboards>) about this update or other tools you want to see in OpenSearch Dashboards.
 
-<img src="{{site.url}}{{site.baseurl}}/images/dashboards/new-look.png" alt="Light and dark mode UI on Discover and Dashboard tools' home page" width="700">
+<img src="{{site.url}}{{site.baseurl}}/images/dashboards/dashboards-home.png" alt="Screenshot of OpenSearch Dashboards homepage" width="700">
 
-{::nomarkdown}<img src="{{site.url}}{{site.baseurl}}/images/icons/alert-icon.png" class="inline-icon" alt="alert icon"/>{:/} **Note**<br>Before you get started, make sure you've installed OpenSearch and OpenSearch Dashboards. For information about installation and configuration, see [Install and configure OpenSearch]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/index/) and [Install and configure OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/install-and-configure/install-dashboards/index/).
-{: .note}
-# Adding sample data
+## What's new
 
-Sample datasets come with visualizations, dashboards, and other tools to help you explore Dashboards before you add your own data. To add sample data, perform the following steps:
+Here's what's new with the homepage:
 
-1. Verify access to OpenSearch Dashboards by connecting to [http://localhost:5601](http://localhost:5601) from a browser. The default username and password are `admin`.
-1. On the OpenSearch Dashboards **Home** page, choose **Add sample data**.
-2. Choose **Add data** to add the datasets, as shown in the following image.
+- **Enhanced user experience:** Improves your experience by incorporating intuitive navigation, streamlined workflows, and a visually appealing interface. 
+- **Optimized data accessibility:** Presents critical information in a more effective manner, making it easier for you to interpret and analyze data.
+- **Responsive design for multi-platform access:** Prioritizes responsive design principles, ensuring seamless access and functionality across various devices and screen sizes. 
```

Comment:
will need to vet this. we don't have any visual regression tests for verifying this. could be accurate now but wrong and we wont know it automatically.

Reply:
@BSFishy Will you review the What's new section to make sure the content is accurate? Thanks!

Reply:
This is accurate. The page extensively uses OUI and properly utilizes the responsive aspects of it, so it should look good on all page sizes. The only disclaimer would be that all within sections are completely owned by plugins, so while the current homepage is responsive, there is not really any guarantee that it is.

Reply:
Given that the new homepage layout will not be default, but rather will be behind a switch, can we adjust the docs accordingly?

Reply:
@wbeckler Are you asking that we put in some text around the new Discover toggle (Data Explorer) needing to be activated by the user?

Reply:
@BSFishy , can you take a look at this and confirm that the homepage plan for 2.12 (keeping current as default and adding a setting to try out the new one) is consistent with these docs changes?

Reply:
@vagimeli The new homepage has its own separate toggle from the Discover/Data Explorer toggle. I think it defaults to old homepage. I'm also not sure if the PR will make it in for 2.12 as it's still Draft: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5613

Reply:
@wbeckler We have specific Discover documentation that I can update when it's ready. this quickstart is intended to cover OpenSearch Dashboards tools generally. Is there any specific information we should remove? Currently, the OpenSearch Playground shows the updates, so I think we need to update the quickstart for relevancy.

Reply:
@kgcreative Will you review the updates at your availability to make sure the UX topics and screenshots are appropriately captured? Thank you.